### PR TITLE
verify there are no packages that require vendoring

### PR DESCRIPTION
**Pull request by mshitrit**
- updated vendored packages
- add verification all packages are vendored

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -300,3 +300,9 @@ container-push: ## Push containers (NOTE: catalog can't be build before bundle w
 .PHONY: test-mutation-ci
 test-mutation-ci: fetch-mutation ## Run mutation tests as part of auto build process.
 	./hack/test-mutation.sh 
+
+.PHONY:verify-vendor
+verify-vendor:
+	go mod tidy
+	go mod vendor
+	./hack/verify-diff.sh
```
Comment (slintes) [inline Makefile:308]: nit: missing new line (it's good practice to let text files end with a new line)

Comment (mshitrit) [reply Makefile]: Fixed :+1:

**Pull request merged**